apollo-mockserver/src/commonMain/kotlin/com/apollographql/apollo3/mockserver/MockServerCommon.kt

```diff
@@ -15,7 +15,7 @@ fun parseHeader(line: String): Pair<String, String> {
   return line.substring(0, index).trim() to line.substring(index + 1, line.length).trim()
 }
 
-class MockRecordedRequest(
+data class MockRecordedRequest(
```

Comment:
Why is this needed ? I'm generally opposed to data classes in public APIs unless there's a strong reason to do so (https://jakewharton.com/public-api-challenges-in-kotlin/).